core/src/util/decimal128.rs

```diff
@@ -0,0 +1,466 @@
+use likely_stable::unlikely;
```

Comment:
unlikely suggests to the branch predictor that a branch is unlikely. I'm not sure if all processors support it, but presumably x86 does.